src/Sulu/Bundle/PreviewBundle/Infrastructure/Symfony/DependencyInjection/Compiler/RegisterPreviewWebspaceClassPass.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Sulu\Bundle\PreviewBundle\Infrastructure\Symfony\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+
+class RegisterPreviewWebspaceClassPass implements CompilerPassInterface
```

Comment:
```suggestion

/**
 * @internal
 */
class RegisterPreviewWebspaceClassPass implements CompilerPassInterface
```

the php cs header is also missing you can also add it via `composer php-cs-fix`

Reply:
added